iocore/cache/Store.cc

```diff
@@ -986,11 +986,11 @@ Store::clear(char *filename, bool clear_dirs)
       }
       for (int b = 0; d->blocks; b++) {
         if (socketManager.pwrite(fd, z, STORE_BLOCK_SIZE, d->offset + (b * STORE_BLOCK_SIZE)) < 0) {
-          close(fd);
+          ink_release_assert(0 == close(fd));
```

Comment:
It would be better to log an error message and/or assert in a debug build.  It is not a good idea to assert on a close() on a production server.

Reply:
This has been in our version of ATS since 5.3. This PR has been made to sync keep the open source in sync